scripts/update.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+# shellcheck disable=1091 # Files should not be followed, but instead manually checked
+set -e
+_SCRIPT_PATH="$(dirname "$(readlink -f -- "$0")")"
+
+export EXTERNAL_PLUGINS='
+https://ci.ender.zone/job/EssentialsX/lastSuccessfulBuild/artifact/*zip*/archive.zip        archive/jars/EssentialsX-*.jar                      Essentials.jar
+https://ci.athion.net/job/FastAsyncWorldEdit/lastSuccessfulBuild/artifact/*zip*/archive.zip archive/artifacts/FastAsyncWorldEdit-Bukkit-*.jar   FastAsyncWorldEdit.jar
+https://nightly.link/GeyserMC/Geyser/workflows/build/master/Geyser-Spigot.zip               Geyser-Spigot.jar                                   Geyser.jar
+https://ci.viaversion.com/job/ViaVersion/lastSuccessfulBuild/artifact/*zip*/archive.zip     archive/build/libs/ViaVersion-*.jar                 ViaVersion.jar
+https://ci.viaversion.com/job/ViaBackwards/lastSuccessfulBuild/artifact/*zip*/archive.zip   archive/build/libs/ViaBackwards-*.jar               ViaBackwards.jar
+https://ci.viaversion.com/job/ViaRewind/lastSuccessfulBuild/artifact/*zip*/archive.zip      archive/build/libs/ViaRewind-*.jar                  ViaRewind.jar
+'
+
+export INTERNAL_PLUGINS='
+https://nightly.link/kaboomserver/commandspy/workflows/main/master/CommandSpy.zip           CommandSpy.jar      CommandSpy.jar
+https://nightly.link/kaboomserver/extras/workflows/main/master/Extras.zip                   Extras.jar          Extras.jar
+https://nightly.link/kaboomserver/icontrolu/workflows/main/master/iControlU.zip             iControlU.jar       iControlU.jar
+https://nightly.link/kaboomserver/particletrails/workflows/main/master/ParticleTrails.zip   ParticleTrails.jar  ParticleTrails.jar
+https://nightly.link/kaboomserver/weapons/workflows/main/master/Weapons.zip                 Weapons.jar         Weapons.jar
+'
```

Comment:
I could try, but unfortunately POSIX sh doesn't support arrays, so I'm not really sure if there's a better way to represent the downloads.

Reply:
I think the best way to do this might be to use `jq` to parse a JSON file containing plugin/server downloads data. I can do that, if that's okay with you.